re_data_ui/src/utils/helpers.ts

```diff
@@ -61,12 +61,14 @@ export const generateAnomalyValue = (anomaly: Anomaly): string => {
     const hours = value / 60 / 60;
     return `${hours.toFixed(2)} hours`;
   }
-  if (anomaly.metric.indexOf('percent') > -1) {
-    return `${value.toFixed(2)}%`;
-  }
-  if (anomaly.metric.indexOf('count') > -1) {
-    return `${value}`;
-  }
+  // console.log('anomaly value -> ', anomaly);
+  // TODO: this line is throwing issues
+  // if (anomaly.metric.indexOf('percent') > -1) {
+  //   return `${value.toFixed(2)}%`;
+  // }
+  // if (anomaly.metric.indexOf('count') > -1) {
+  //   return `${value}`;
+  // }
```

Comment:
there's likely no need for `generateAnomalyValue` function again. Same with `generateAnomalyMessage`.